integration-cli/docker_cli_build_test.go

```diff
@@ -397,30 +399,30 @@ func (s *DockerSuite) TestBuildLastModified(c *check.C) {
 ADD %s/file /`
```

Comment:
Out of scope: This is kind of dangerous and should be defined as a closure instead.